uarray/_uarray_dispatch.cxx

```diff
@@ -278,10 +278,10 @@ PyObject * clear_backends(PyObject * /* self */, PyObject * args)
 template <typename T>
 class context_helper
 {
-  T new_backend_;
-  std::vector<T> * backends_;
   std::vector<size_t> sizes_;
 public:
+  T new_backend_;
```

Comment:
Can you at least make this read-only, i.e. add a member function `const T & get_backend() const;`.